schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -3414,7 +3414,7 @@ FanSystem:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: False
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: False
```

Comment:
If this is set by ventilation requirements then I think it should be the same across the rotations, agreed?

Reply:
Yes but not if it is set by minimum damper position